lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -243,7 +242,7 @@ def create_post_for_auth_or_purchase(money, payment, options)
           add_customer_data(post, options)
           add_metadata(post, options)
           post[:description] = options[:description]
-          post[:statement_description] = options[:statement_description]
+          post[:statement_descriptor] = options[:statement_description]
```

Comment:
:cool:

You narrowed in on the exact option that triggered this pull request.  :+1: 

For old Stripe API versions, `statement_description` worked great.  For newer versions, it bombs with `Received unknown parameter: statement_description`.

So with this PR, we explicitly specify the API version which means we don't really care what the underlying Stripe account is specifying version wise.  We're saying what version we're wanting.